client/components/date-picker/content.js

```diff
@@ -89,26 +93,55 @@ class DatePickerContent extends Component {
 										inValidDays="future"
 									/>
 								) }
-								<H className="woocommerce-date-picker__text">{ __( 'compare to', 'wc-admin' ) }</H>
-								<ComparePeriods onSelect={ onSelect } compare={ compare } />
-								{ isValidSelection( selectedTab ) ? (
-									<Link
-										className="woocommerce-date-picker__update-btn components-button is-button is-primary"
-										to={ getUpdatePath( selectedTab ) }
-										onClick={ onClose }
-									>
-										{ __( 'Update', 'wc-admin' ) }
-									</Link>
-								) : (
-									<Button className="woocommerce-date-picker__update-btn" isPrimary disabled>
-										{ __( 'Update', 'wc-admin' ) }
-									</Button>
-								) }
+								<div
+									className={ classnames( 'woocommerce-date-picker__content-controls', {
+										'is-sticky-bottom': selectedTab === 'custom' && isMobileViewport(),
+										'is-custom': selectedTab === 'custom',
+									} ) }
+								>
+									<H className="woocommerce-date-picker__text">
+										{ __( 'compare to', 'wc-admin' ) }
+									</H>
+									<ComparePeriods onSelect={ onSelect } compare={ compare } />
+									<div className="woocommerce-date-picker__content-controls-btns">
+										{ selectedTab === 'custom' && (
+											<Button
+												className="woocommerce-date-picker__content-controls-btn"
+												isPrimary
+												onClick={ resetCustomValues }
+												disabled={ ! ( after || before ) }
+											>
+												{ __( 'Reset', 'wc-admin' ) }
+											</Button>
+										) }
+										{ isValidSelection( selectedTab ) ? (
+											<Link
+												className={ classnames(
```

Comment:
If all the classNames are static, you can just pass them in as a string rather than using `classNames()`